practice/practice-one/src/index.html

```diff
@@ -190,7 +190,209 @@ <h4>Front-end Developer Apple .Inc</h4>
       </div>
     </section>
     <!--end section experience-->
+
+    <!--section blog-->
+    <section class="section-blog">
+      <div class="container">
+        <div class="title-blog">
+          <p>My articles</p>
+          <h4>Personal Blog</h4>
+        </div>
+        <div class="post-item">
+          <div class="card">
+            <img src="./img/blog-item-1.png" alt="picture blog item">
+            <div class="card-title">How to prototype in figma</div>
+            <div class="time">July 15, 2021</div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              Fusce id magna nec ante scelerisque laoreet. Fusce finibus ...
+            </p>
+            <a href="#">Continue Reading</a>
```

Comment:
Currently, we should use `javascript(void(0))` to avoid redirecting to the top of the page.